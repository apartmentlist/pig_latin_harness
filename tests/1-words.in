cat
apple
school
Pig
Each
yellow
rhythm
