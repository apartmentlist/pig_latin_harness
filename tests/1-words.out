atcay
appleway
oolschay
Igpay
Eachway
ellowyay
ythmrhay
